src/source/Rtp/RtpPacket.c

```diff
@@ -246,9 +261,13 @@ STATUS setBytesFromRtpPacket(PRtpPacket pRtpPacket, PBYTE pRawPacket, UINT32 pac
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
     PRtpPacketHeader pHeader = &pRtpPacket->header;
-    UINT32 packetLengthNeeded = 0;
-    PBYTE pCurPtr = pRawPacket;
-    UINT8 i;
+    size_t packetLengthNeeded = 0;
```

Comment:
Use `SIZE_T`

Reply:
I used SIZE_T earlier but seem like in PIC the correct type for macOS is not defined. Hence it was failing in MAC CI builds
error: incompatible pointer types passing 'SIZE_T *' (aka 'unsigned long long *') to parameter of type 'size_t *' (aka 'unsigned long *')

Reply:
Ah I see the problem. On MacOS `size_t` is `unsigned long long` on gcc and `unsigned long` with clang compiler.  Even though the failing job says in the title gcc, if you look at the output it's actually using clang.  We map SIZE_T to unsigned long long , hence it if any of the following macros are defined

```
#if defined __GNUC__ || defined __GNUG__
#if defined __x86_64__ || defined __ppc64__ || __aarch64__
```

clang also defines `__GNUC__`, I will fix that and post the update, I'm surprised this did not fail in PIC tests.

Reply:
Should we merge this PR? Or wait for your fix?

Reply:
Yeah let's not block this, looks good, I've approved.